src/sssom/parsers.py

```diff
@@ -643,7 +640,9 @@ def from_obographs(
                                     mdict[SUBJECT_LABEL] = label
                                     mdict[PREDICATE_ID] = "oboInOwl:hasDbXref"
                                     mdict[MAPPING_JUSTIFICATION] = MAPPING_JUSTIFICATION_UNSPECIFIED
-                                    mlist.append(Mapping(**mdict))
+                                    mapping = _ensure_valid_mapping_from_dict(mdict)
```

Comment:
can you use `_add_valid_mapping_to_list` here?

Reply:
Oversight, done.